Breadcrumbs/Breadcrumbs.vue

```diff
@@ -4,6 +4,24 @@ import { config } from '~/config'
 // Constants
 import { BUTTON_PRESET } from '~/components/Button/constants/button-preset.constant'
 
+const props = withDefaults(
```

Comment:
Pls don't come up with new ways to define props, we use a different approach throughout the app